### PR TITLE
Cargo outfit + small changes to item collection

### PR DESCRIPTION
**Pull request by ShiftyRail**
What is says on the title

### CODE REVIEW

File: code/datums/outfit/cargo.dm [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+/datum/outfit/hop
+
+	outfit_name = "Head of Personnel"
+	associated_job = /datum/job/hop
+
+	backpack_types = list(
+		BACKPACK_STRING = /obj/item/weapon/storage/backpack,
+		SATCHEL_NORM_STRING = /obj/item/weapon/storage/backpack/satchel_norm,
+		SATCHEL_ALT_STRING = /obj/item/weapon/storage/backpack/satchel,
+		MESSENGER_BAG_STRING = /obj/item/weapon/storage/backpack,
```
Comment (jknpj) [inline code/datums/outfit/cargo.dm]: :thinking:

Comment (ShiftyRail) [reply code/datums/outfit/cargo.dm]: :face_with_head_bandage:

**Comment by utilitas-** [comment]
where the fuck did cargo shorts go?

**Comment by ShiftyRail** [comment]
`/obj/item/clothing/under/rank/cargotech` are the cargo shorts

**Comment by utilitas-** [comment]
when did they get removed from the lockers and from the default loadout of cargo techs? they are a lost artifact now; forever lost to time (and mappers)

**Comment by ShiftyRail** [comment]
The "cargotech" jumpsuit has been the default uniform for cargo techies since at least 7 years ago.
I played a Cargo Technician yesterday, the icon does look like shorts.

**Comment by utilitas-** [comment]
I swear to god it looked like shorts
this is some serious berenstain bears bullshit

**Pull request merged**